components/Typography/src/UIFontDescriptor+MaterialTypography.m

```diff
@@ -36,24 +36,32 @@ + (nonnull UIFontDescriptor *)mdc_preferredFontDescriptorForMaterialTextStyle:
   // Store the system font family name to ensure that we load the system font.
   // If we do not explicitly include this UIFontDescriptorFamilyAttribute in the
   // FontDescriptor the OS will default to Helvetica.
```

Comment:
Nit: Could you please update this to include iOS 9+ with San Francisco?  Your PR discussion was really good, so something like, "On iOS 9+, the Font Family changes from San Francisco to San Francisco Display at point size 20".  You could probably put the entire comment together and remove the one-liner just below about font size 20+.